readthedocs/core/mixins.py

```diff
@@ -64,6 +65,11 @@ def can_be_cached(self, request):
 
     @lru_cache(maxsize=1)
     def _is_cache_enabled(self, project):
-        """Helper function to check if CND is enabled for a project."""
-        # TODO: check for the organization's plan.
-        return settings.ALLOW_PRIVATE_REPOS and project.has_feature(Feature.CDN_ENABLED)
+        """Helper function to check if CDN is enabled for a project."""
+        plan_has_cdn = PlanFeature.objects.filter(
+            feature_type=PlanFeature.TYPE_CDN,
+            plan__subscriptions__organization__in=project.organizations.all(),
+        ).exists()
```

Comment:
you can use `PlanFeature.get_feature`
https://github.com/readthedocs/readthedocs.org/blob/2615c5f72d11919a4dac740db7606baa28d53488/readthedocs/subscriptions/managers.py#L197-L197

it returns None if the project doesn't have the feature.